### PR TITLE
ElPro Update profile.md

### PR DESCRIPTION
**Pull request by ElProgreso**
I hope I got it right :))

### CODE REVIEW

File: governance/delegates/0xAF8aa6846539033Eaf0c3ca4C9C7373e370E039b/profile.md [hunk 1]
```diff
@@ -1,52 +1,45 @@
...
 
 ### Key Info
 
-Delegate Address: 0xEtcetera  
-Forum: @LongForWisdom  
-Rocketchat: @LongForWisdom  
-Email (optional): longforwisdomandstuff@gmail.com  
-External Website (optional): www.longforwisdom.com  
-Meet Your Delegate Video (optional): youtube or something  
+Delegate Address: flapper.eth
```
Comment (LongForWisdom) [inline governance/delegates/0xAF8aa6846539033Eaf0c3ca4C9C7373e370E039b/profile.md:11]: I don't think this points to your contract?

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
Not sure address is correct.

**Comment by LongForWisdom** [comment]
Pretty sure you can delete the img in this PR as well.

File: governance/delegates/0xAF8aa6846539033Eaf0c3ca4C9C7373e370E039b/profile.md (hunk 1 above)
Comment (ElProgreso) [reply governance/delegates/0xAF8aa6846539033Eaf0c3ca4C9C7373e370E039b/profile.md:11]: It should point now to my contract

**Review by ElProgreso — COMMENTED** [review]
I pointed flapper.eth to contract 0xAF8aa6846539033Eaf0c3ca4C9C7373e370E039b

**Review by LongForWisdom — APPROVED** [review]

**Pull request merged**